src/library_dylink.js

```diff
@@ -654,6 +654,14 @@ var LibraryDylink = {
               __ATINIT__.push(init);
             }
           }
+          var apply_relocs = moduleExports['__wasm_apply_data_relocs'];
```

Comment:
I think this would be more consistent:
```suggestion
          var applyRelocs = moduleExports['__wasm_apply_data_relocs'];
```